### PR TITLE
TC_04.001.04 | Folder > Rename Folder| Verify to rename the folder from drop-down menu of the folder element in the breadcrumbs.

### PR DESCRIPTION
**Pull request by Ridgewood2016**
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/25

### CODE REVIEW

**Review by YanaLifanova — COMMENTED** [review]
describe need to have US number.In comments must to be a link on US. It must to contain number as well

File: cypress/e2e/renameFolder.cy.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+describe('Folder > Rename Folder', () => {
+    it('Verify to rename the folder from drop-down menu of the folder element in the breadcrumbs', () => {
+        cy.get('span').contains('New Item').click()
+        cy.wait(1000)
+        cy.get('input[name="name"]').clear()
+        cy.get('input[name="name"]').type('Folder')
+        cy.get('.com_cloudbees_hudson_plugins_folder_Folder').click()
+        cy.get('button').contains('OK').click()
+        cy.get('button').contains('Save').click()
+        cy.get('div[id="breadcrumbBar"]').contains('Folder')
```
Comment (wwwser11) [inline cypress/e2e/renameFolder.cy.js:16]: What does this string do?

**Comment by Ridgewood2016** [comment]
https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/25

File: cypress/e2e/renameFolder.cy.js (hunk 1 above)
Comment (Ridgewood2016) [reply cypress/e2e/renameFolder.cy.js]: select an item type: Folder

File: cypress/e2e/folderRenameFolder.cy.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/// <reference types="cypress"/>
+const USERNAME = Cypress.env('local.admin.username');
+const PASSWORD = Cypress.env('local.admin.password');
```
Comment (Negrescu29) [inline cypress/e2e/folderRenameFolder.cy.js:3]: As mentioned in another PR this 2 variables are not used in this test.

Comment (Ridgewood2016) [reply cypress/e2e/folderRenameFolder.cy.js]: @Negrescu29 Fixed , please approve

**Review by Negrescu29 — APPROVED** [review]
👍🏼

**Review by NShakiro — APPROVED** [review]

**Pull request merged**